momento-sdk/src/main/java/momento/sdk/config/Configurations.java

```diff
@@ -79,6 +99,16 @@ private LowLatency(TransportStrategy transportStrategy) {
      * @return the latest low-latency configuration
      */
     public static Configuration Latest() {
+      return LowLatency.V1();
+    }
+
+    /**
+     * Provides v1 configuration for a low-latency environment. This configuration is guaranteed not
+     * to change in future releases of the Momento Java SDK.
+     *
+     * @return the v1 low-latency configuration
+     */
+    public static Configuration V1() {
```

Comment:
nit for all of these - I think the `PascalCasing` was for .NET.  It might be more idiomatic to use `camelCasing` for these in Java, but not a big deal

Reply:
Good catch.